helix-core/src/main/java/org/apache/helix/util/StatusUpdateUtil.java

```diff
@@ -55,6 +56,12 @@
 public class StatusUpdateUtil {
   static Logger _logger = LoggerFactory.getLogger(StatusUpdateUtil.class);
 
+  public static final boolean ERROR_LOG_TO_ZK_ENABLED;
+  static {
+    String valueString = System.getProperty(SystemPropertyKeys.STATEUPDATEUTIL_ERROR_LOG_ENABLED, "");
+    ERROR_LOG_TO_ZK_ENABLED = valueString.equals("enabled") ? true : false;
```

Comment:
Please use HelixUtil.getSystemPropertyAsBoolean(), it allows you to specify the default value there in the method.

Reply:
There is only HelixUtil.getSystemPropertyAsInt/Long, no asBoolean(). Let me add as Boolean then, make the changes a little bit wider.